frontend/src/scenes/billing/BillingProductAddon.tsx

```diff
@@ -140,6 +171,36 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
                                     }
                                 />
                             )
+                        ) : billing?.trial?.target === addon.type ? (
+                            <div className="flex flex-col items-end justify-end">
+                                <Tooltip
+                                    title={
+                                        <p>
+                                            You are currently on a free trial for{' '}
+                                            <b>{toSentenceCase(billing.trial.target)}</b> until{' '}
+                                            <b>{dayjs(billing.trial.expires_at).format('LL')}</b>. At the end of the
+                                            trial{' '}
+                                            {billing.trial.type === 'autosubscribe'
+                                                ? 'you will be automatically subscribed to the plan.'
+                                                : 'you will be asked to subscribe. If you choose not to, you will lose access to the features.'}
+                                        </p>
+                                    }
+                                >
+                                    <LemonTag type="completion" icon={<IconCheckCircle />}>
+                                        You're on a trial for this add-on
+                                    </LemonTag>
+                                </Tooltip>
+                                {/* Comment out until we can make sure a customer can't activate a trial multiple times */}
+                                {/* <LemonButton
+                                    type="primary"
+                                    size="small"
+                                    onClick={cancelTrial}
+                                    loading={trialLoading}
+                                    className="mt-1"
+                                >
+                                    Cancel trial
+                                </LemonButton> */}
```

Comment:
Should we at least give them a button to contact support to cancel?

Or just make sure people can't activate trial multiple times in the next couple of days.. it would be very weird to not have any way to cancel, after we say they can cancel it at any time

Reply:
Yeah it's more of mental block tbh, I just want to get this out and I'll think through cancel tomorrow.